synapse/models/infotech.py

```diff
@@ -1076,20 +1083,46 @@ async def _onFormItDevStr(node):
                 'doc': 'The software creates the file path.'}),
 
             (('it:software', 'creates', 'it:os:windows:registry:entry'), {
-                'doc': 'The software creates the windows registry entry.'}),
+                'doc': 'The software creates the Microsoft Windows registry entry.'}),
+
+            (('it:software', 'creates', 'it:os:windows:service'), {
+                'doc': 'The software creates the Microsoft Windows service.'}),
 
             (('it:exec:query', 'found', None), {
                 'doc': 'The target node was returned as a result of running the query.'}),
 
-            # FIXME tighten these down...
-            (('it:app:snort:rule', 'detects', None), {
-                'doc': 'The snort rule is intended for use in detecting the target node.'}),
+            (('it:app:snort:rule', 'detects', 'risk:vuln'), {
```

Comment:
Snort rules (much like YARA rules) can detect arbitrary things, such as "that one FQDN I decided was bad" to "suspicious HTTP request", etc.  Are we making these targets too restrictive?

Reply:
Attempting to eliminate open-ended edges but happy to discuss ( though the PR is already merged )